traffic_ops/traffic_ops_golang/login/login.go

```diff
@@ -232,6 +232,13 @@ func TokenLoginHandler(db *sqlx.DB, cfg config.Config) http.HandlerFunc {
 			return
 		}
 
+		_, dbErr := db.Exec(UpdateLoginTimeQuery, username)
+		if dbErr != nil {
+			log.Errorf("unable to update authentication time for a given user: %s\n", dbErr.Error())
```

Comment:
Actually, disregard, because it looks like you're double-logging this, because `api.HandleErr` will log it too. It looks like what you actually want something like:
```go
if dbErr != nil {
    dbErr = fmt.Errorf("unable to update authentication time for user '%s': %w", username, dbErr)
    api.HandleErr(w, r, nil, http.StatusInternalServerError, nil, dbErr)
    return
}
```
(I think the username is helpful information)